packages/monaco/src/browser/monaco-workspace.ts

```diff
@@ -147,12 +147,12 @@ export class MonacoWorkspace extends BaseMonacoWorkspace implements lang.Workspa
             promises.push(this.monacoModelResolver.createModelReference(edit.resource).then(reference => {
                 const model = reference.object.textEditorModel;
                 const range = edit.range;
-                model.applyEdits([{
+                model.pushEditOperations([], [{
```

Comment:
or `openerService.open(model.uri, { revealIfVisible: false, selection }).then(widget => if(widget instaceof MonacoEditor) {...})` then it will open an editor if there is no, actually it feels that vscode does like that, does not access the model directly